pkg/services/authn/identity.go

```diff
@@ -71,6 +71,8 @@ type Identity struct {
 	// IDToken is a signed token representing the identity that can be forwarded to plugins and external services.
 	// Will only be set when featuremgmt.FlagIdForwarding is enabled.
 	IDToken string
+	// UserUID is the unique identifier for the entity in the Grafana database.
+	UserUID string
```

Comment:
Same here we should rename to UID instead

Reply:
I left this as UserUID for now since that was an existing field name in SignedInUser